config.md

```diff
@@ -69,6 +69,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
 
 * **`destination`** (string, REQUIRED) Destination of mount point: path inside container.
     This value MUST be an absolute path.
+    * Linux: It MAY be a relative path too, in which case it MUST be relative to "/".
```

Comment:
Perhaps the combination of `MUST` and `MAY` here is a bit ambiguous.

As we're differentiating per platform here, perhaps we should

- make each of these a "complete" requirement 🤔 
- For Linux, I would still consider it to be a `SHOULD` (strong recommendation), as that's what we'd preferred it to have been (were it not for backward compatibility).
- If we only define "per platform", I guess we need to add a "For other platforms .... " which would be a `MUST` (unless we know of implementations on other platforms that use relative paths)

Also wondering if it would be worth mentioning _why_ we allow (`MAY`) have relative paths on Linux; maybe it doesn't hurt to explain the reason (i.e. "we tried, but had to continue it for backward compatibility"). Not sure if we've had similar cases before, and if there's wording we could use for such cases.

Reply:
Makes sense, will update it tomorrow. Thanks! :)

Reply:
@thaJeztah Does it looks good now? I was not decided between this version I pushed and another one saying just "This value MUST be an absolute path, except when the platform listed below say otherwise", and then just Linux says otherwise.

Let me know what you think :)